client/state/rewind/selectors/get-backup-retention-update-status.ts

```diff
@@ -0,0 +1,13 @@
+import type { AppState } from 'calypso/types';
+
+/**
+ * Returns the update backup retention request's status.
+ *
+ * @param state The application state.
+ * @param siteId The site ID for which to retrieve days of backups saved.
+ * @returns The the status of the request.
+ */
+const getBackupRetentionUpdateRequestStatus = ( state: AppState, siteId: number ): string =>
+	state.rewind[ siteId ]?.retention?.updateBackupRetentionRequestStatus ?? 'unsubmitted';
```

Comment:
Just like the rest of the states of this reducer are suggested to be constants, `unsubmitted` should likely be one of those constants.